### PR TITLE
Complete  thursday jan 30th 1908

### PR DESCRIPTION
**Pull request by jbh18g**
Vade Mecum for the homeward bound not in templates.  All templates in place, mostly unedited.

### CODE REVIEW

File: 1908-01-30.xml [hunk 1]
```diff
@@ -1923,28 +1923,5053 @@
                 </div>
             </div>
             <pb n="3"/>
-            <div type="page" n="3"> 
-            
+            <div type="page" n="3">
+                <cols n="6"/>
+                <cb n="1"/>
+                <div type="feature" feature="local">
```
Comment (huoyanzhizhou) [inline 1908-01-30.xml]: There is no div type called feature. Instead, try div type="section"

File: 1908-01-30.xml [hunk 2]
```diff
@@ -1923,28 +1923,5053 @@
...
+                        were naturally not as fast a s a hard course</p>
+                    <p>Sir Eldos aod Lady Gorst were present and the attendance was decidedly
+                        good.</p>
+                    <p>The fields in some of the race dwindleri down rather, but we witnessed some
+                        very good finishes in the first race the starter pony broke loome and nosed
+                        &amp; little diversion Nleyban ran a good noe, this is his second win this
+                        season on this coure He is evidently</p>
+                    <p>good pony - Add we wish his sporting owners further lack with him. The Grand
+                        National Steeplechase was most interesting We are sorry to hear that Capt.
+                        Drake di
```
Comment (huoyanzhizhou) [inline 1908-01-30.xml]: I can find many dates back to 1903 and 1904. Please make sure that you update your meatures.

**Comment by huoyanzhizhou** [comment]
Please check your cols and cb, it seems that you lack some of them.

**Comment by huoyanzhizhou** [comment]
You new file has not been updated, please keep updating.

**Comment by huoyanzhizhou** [comment]
Overall good job. Although there are some measures needed to be updated, we can consider merging this one, @whanley. Also, sorry for the last comment, that was my mistake and I have deleted that.

**Comment by whanley** [comment]
Looks great @jbh18g . Note that the day of the week is wrong in your Saturday header.

**Pull request merged**